configs/setup_golden_image.py

```diff
@@ -183,23 +169,24 @@ def main(args):
     assert os.geteuid() == 0, 'This script requires root permissions'
 
     # -- Build 'raw' golden image using vmdebootstrap --
+    # Skip bootstrap when --image param is used
     if not getattr(args, 'skip_bootstrap', False):
-        #vmd_path = args.extra[1] if len(args.extra) >= 2 else None
-        vmd_path = getattr(args, 'vmd_cfg', None)
-        debootstrap_image(args.config, vmd_path=vmd_path)
+        if getattr(args, 'image', None) is None:
+            return
+            vmd_path = getattr(args, 'vmd_cfg', None)
```

Comment:
This indentation is probably broken logic, does it need to shift left?

And absolutely return the exact reason.  You used two "if" statements, use two error explicit and precise error messages.